site/.vitepress/config.ts

```diff
@@ -519,20 +505,20 @@ export default defineConfig({
         base: "/packages/aa-accounts",
         items: [
           {
-            text: "Overview of aa-accounts",
+            text: "Getting Started",
             link: "/",
           },
           {
-            text: "LightSmartContractAccount",
```

Comment:
I'd keep this name because it's the actual class name. wdyt?

Reply:
yeah, but we use just `Light Account` in most of our doc, so thought would be clearer to make the term consistent. I think it's self-explanatory.